BunnymodXT/discord_integration.cpp

```diff
@@ -0,0 +1,338 @@
+									presence.largeImageText = gmc_thumbnail_to_chapter.find(presence.largeImageKey)->second.data();
+								}
+							}
+
+							else if (!strncmp(gameDir, "czeror", 6))
+							{
+								if (czds_map_name_to_thumbnail.find(map_name) != czds_map_name_to_thumbnail.cend())
+								{
+									presence.largeImageKey = czds_map_name_to_thumbnail.find(map_name)->second.data();
+									presence.largeImageText = czds_thumbnail_to_chapter.find(presence.largeImageKey)->second.data();
+								}
+							}
+
+							else if (!strncmp(gameDir, "wantedsp", 8))
+							{
+								if (wanted_map_name_to_thumbnail.find(map_name) != wanted_map_name_to_thumbnail.cend())
+								{
+									presence.largeImageKey = wanted_map_name_to_thumbnail.find(map_name)->second.data();
+									presence.largeImageText = wanted_thumbnail_to_chapter.find(presence.largeImageKey)->second.data();
+								}
+							}
+
+							else if (!strncmp(gameDir, "echoes", 6))
+							{
+								if (echoes_map_name_to_thumbnail.find(map_name) != echoes_map_name_to_thumbnail.cend())
+								{
+									presence.largeImageKey = echoes_map_name_to_thumbnail.find(map_name)->second.data();
+									presence.largeImageText = echoes_thumbnail_to_chapter.find(presence.largeImageKey)->second.data();
+								}
+							}
+
+							else if (!strncmp(gameDir, "caged_fgs", 9))
+							{
+								if (caged_map_name_to_thumbnail.find(map_name) != caged_map_name_to_thumbnail.cend())
+								{
+									presence.largeImageKey = caged_map_name_to_thumbnail.find(map_name)->second.data();
+									presence.largeImageText = caged_thumbnail_to_chapter.find(presence.largeImageKey)->second.data();
+								}
+							}
+
+							else if (!strncmp(gameDir, "poke646", 7))
+							{
+								if (poke646_map_name_to_thumbnail.find(map_name) != poke646_map_name_to_thumbnail.cend())
+								{
+									presence.largeImageKey = poke646_map_name_to_thumbnail.find(map_name)->second.data();
+									presence.largeImageText = poke646_thumbnail_to_chapter.find(presence.largeImageKey)->second.data();
+								}
+							}
+						}
+					}
+				}
+
+				char buffer_state[128];
+				char buffer_stop[128];
+
+				const auto& gt = CustomHud::GetTime();
+
+				const auto current_timestamp = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch()).count();
+				start_timestamp = current_timestamp - total_time;
+
+				sprintf(buffer_state, "%s | FPS Max: %g | Skill: %u", state.c_str(), CVars::fps_max.GetFloat(), CVars::skill.GetInt());
+				presence.state = buffer_state;
+
+				if (CustomHud::GetCountingTime())
+				{
+					if (CustomHud::GetInvalidRun()) {
+						presence.smallImageKey = "discord_pink";
+						presence.smallImageText = "BXT prevented the crash, so that run is no longer valid!";
+					} else {
+						presence.smallImageKey = "discord_green";
+						presence.smallImageText = "BXT timer is activated. Good luck for the run!";
```

Comment:
```suggestion
						presence.smallImageText = "Run in progress";
```
the whole "BXT timer is activated. Good luck for the run!" is too long imo and good luck is unnecessary in the discord status

Reply:
Done.